docs/api-guide.md

```diff
@@ -216,177 +237,219 @@ Set<String> namespaces = admin.getNamespaceNames();
 transactionManager.close();
 ```
 
-### Begin/Start a transaction
+### Begin or start a transaction
+
+Before executing transactional CRUD operations, you need to begin or start a transaction.
 
-You need to begin/start a transaction before executing transactional CRUD operations.
-You can begin/start a transaction as follows:
+You can begin a transaction as follows:
 
 ```java
-// Begin a transaction
+// Begin a transaction.
 DistributedTransaction transaction = transactionManager.begin();
+```
 
-Or
+Or, you can start a transaction as follows:
 
-// Start a transaction
+```java
+// Start a transaction.
 DistributedTransaction transaction = transactionManager.start();
 ```
 
-You can also begin/start a transaction with specifying a transaction ID as follows:
+Alternatively, you can use the `begin` method for a transaction by specifying a transaction ID as follows:
 
 ```java
-// Begin a transaction with specifying a transaction ID
-DistributedTransaction transaction = transactionManager.begin("<transaction ID>");
+// Begin a transaction with specifying a transaction ID.
+DistributedTransaction transaction = transactionManager.begin("<TRANSACTION_ID>");
+```
 
-Or
+Or, you can use the `start` method for a transaction by specifying a transaction ID as follows:
 
-// Start a transaction with specifying a transaction ID
-DistributedTransaction transaction = transactionManager.start("<transaction ID>");
+```java
+// Start a transaction with specifying a transaction ID.
+DistributedTransaction transaction = transactionManager.start("<TRANSACTION_ID>");
 ```
 
-Note that you must guarantee uniqueness of the transaction ID in this case.
+{% capture notice--info %}
+**Note**
+
+Specifying a transaction ID is useful when you want to link external systems to ScalarDB. Otherwise, you should use the `start()` method.
+
+When you specify a transaction ID, make sure you specify a unique ID (for example, UUID v4) throughout the system since ScalarDB depends on the uniqueness of transaction IDs for correctness.
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
 ### Join a transaction
 
-You can join an ongoing transaction that has already begun with specifying a transaction ID as follows:
+Joining a transaction is particularly useful in a stateful application where a transaction spans multiple client requests. In such a scenario, the application can start a transaction during the first client request. Then, in subsequent client requests, the application can join the ongoing transaction by using the `join()` method.
+
+You can join an ongoing transaction that has already begun by specifying the transaction ID as follows:
+
+```java
+// Join a transaction.
+DistributedTransaction transaction = transactionManager.join("<TRANSACTION_ID>");
+```
+
+{% capture notice--info %}
+**Note**
+
+You must guarantee the uniqueness of the transaction ID in this case. To get the transaction ID with `getId()`, you can specify the following:
```

Comment:
The first sentence is unnecessary as users don’t need to care about the uniqueness of the transaction ID here; they can simply get the transaction ID with `getId()` and use it.
```suggestion
To get the transaction ID with `getId()`, you can specify the following:
```

Reply:
Got it. Thanks!